core/arch/arm/plat-imx/imx.h

```diff
@@ -39,7 +39,5 @@ void imx_gpcv2_set_core_pgc(bool enable, uint32_t offset);
 
 #ifdef CFG_TZC380
 void imx_configure_tzasc(void);
-#else
-static inline void imx_configure_tzasc(void) {}
```

Comment:
Why?

Reply:
It is just my preference. I can revert it since this change is not necessary.

Reply:
Yes please. The pattern `#if CFG_FOO // func prototype // #else // static inline func {} // #endif` is preferred instead of having `#ifdefs` in the `.c` files.

Reply:
Alright, no problem!